test/plugins/pipelines.test.js

```diff
@@ -2009,8 +2009,6 @@ describe('pipeline plugin test', () => {
 
         it('returns 200 and updates settings only', () => {
             const expectedSetting = {
-                showEventTriggers: false,
```

Comment:
Some tests were failing in the following PRs and have been corrected
https://github.com/screwdriver-cd/data-schema/pull/484